.github/workflows/docker_image_build.yaml

```diff
@@ -0,0 +1,22 @@
+name: Building the Docker image containing the basic tools for building ParallelZone
+on:
+  push:
```

Comment:
If you want to add a manual way of running the action you can add `workflow_dispatch`, i.e.:
```
on:
  workflow_dispatch:
  push:
```

Reply:
Thanks for letting me know about this trick!